pkg/commands/build/status_test.go

```diff
@@ -77,7 +77,6 @@ bp-id-2         bp-version-2         mysupercoolsite2.com
 		clientSetProvider := testhelpers.GetFakeKpackProvider(clientSet, defaultNamespace)
 
 		fakeFetcher := registryfakes.Fetcher{}
```

Comment:
the fake fetcher and other objects related to registry access can be removed in the test

Reply:
Please see 831e0496cbc9a6ee8da4e5a997a5487d6c0a8714.